gapic/api/artman_errorreporting.yaml

```diff
@@ -0,0 +1,28 @@
+common:
+  api_name: google-devtools-clouderrorreporting-v1beta1
+  import_proto_path:
+    - ${THISDIR}/../..
+  src_proto_path:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/v1beta1
+  service_yaml:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/errorreporting.yaml
+  gapic_api_yaml:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/v1beta1/clouderrorreporting_gapic.yaml
```

Comment:
Oh, sorry, I thought you were commenting to the filename of the file itself. Sorry. And you are right.